parser/SqlParser.ypp

```diff
@@ -834,23 +846,34 @@ key_value:
   | key_string_list {
     $$ = $1;
   }
-  | key_literal_value {
+  | key_integer_value {
     $$ = $1;
   };
 
 key_string_value:
   any_name any_name {
     $$ = new quickstep::ParseKeyStringValue(@1.first_line, @1.first_column, $1, $2);
-  };
+  }
+  | any_name TOKEN_ALL {
+    // This is a special case to handle the COMPRESS ALL option of the BLOCK PROPERTIES.
+    $$ = new quickstep::ParseKeyStringValue(@1.first_line, @1.first_column, $1, 
+        new quickstep::ParseString(@2.first_line, @2.first_column, "ALL"));
+  }
 
 key_string_list:
   any_name '(' name_commalist ')' {
     $$ = new quickstep::ParseKeyStringList(@1.first_line, @1.first_column, $1, $3);
   };
 
-key_literal_value:
-  any_name literal_value {
-    $$ = new quickstep::ParseKeyLiteralValue(@1.first_line, @1.first_column, $1, $2);
+key_integer_value:
+  any_name TOKEN_UNSIGNED_NUMVAL {
+    if ($2->float_like()) {
```

Comment:
@cramja We can file an issue for the new tokens, so that we can consider it as a separate PR.